cli/src/cli_builder.rs

```diff
@@ -577,8 +577,8 @@ fn xxx<'a>(command: CliCommand) -> Option<&'a str> {
         CliCommand::ProbeView => Some("experimental probe view"),
 
         // Metrics-related subcommands
-        CliCommand::TimeseriesQuery => Some("experimental timeseries query"),
-        CliCommand::TimeseriesSchemaList => Some("experimental timeseries schema list"),
+        CliCommand::SystemTimeseriesQuery => Some("experimental timeseries query"),
+        CliCommand::SystemTimeseriesSchemaList => Some("experimental timeseries schema list"),
```

Comment:
Leaning against adding `system` here — it's already experimental and I'm not sure we're going to bother adding the project-scoped one from https://github.com/oxidecomputer/omicron/pull/6873 when it lands.